app/settings.py

```diff
@@ -55,7 +55,10 @@
                 'social_django.context_processors.backends',
                 'social_django.context_processors.login_redirect',
             ],
-            'builtins': ['django.contrib.staticfiles.templatetags.staticfiles']
```

Comment:
Through this piece of code I've created a custom filter for replacing space with underscore for Wikipedia article names.